app.rb

```diff
@@ -1,7 +1,32 @@
 require 'sinatra/base'
+require './lib/Player'
+require './lib/play_game'
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  use Rack::Session::Cookie, :key => 'rack.session', :path => '/', :secret => 'abcd'
```

Comment:
just out of curiosity, what does this line do?

Reply:
it's instead of enable :session- an alternative to it that seemed to work